SPECS/kubevirt/kubevirt.spec

```diff
@@ -95,20 +116,111 @@ Group:          System/Packages
 %description    virt-operator
 The virt-opertor package provides an operator for kubevirt CRD
 
+%package        pr-helper-conf
+Summary:        Configuration files for persistent reservation helper
+Group:          System/Packages
+
+%description    pr-helper-conf
+The pr-helper-conf package provides configuration files for persistent
+reservation helper
+
+%package        manifests
+Summary:        YAML manifests used to install kubevirt
+Group:          System/Packages
+
+%description    manifests
+This contains the built YAML manifests used to install kubevirt into a
+kubernetes installation with kubectl apply.
+
 %package        tests
 Summary:        Kubevirt functional tests
 Group:          System/Packages
 
 %description    tests
 The package provides Kubevirt end-to-end tests.
 
+%package -n     obs-service-kubevirt_containers_meta
+Summary:        Kubevirt containers meta information (build service)
+Group:          System/Packages
+
+%description -n obs-service-kubevirt_containers_meta
```

Comment:
We don't need the containers_meta packages (and associated files) for mariner. Suggest removing it and add a comment here instead so future upgrades can refer it.